website/src/parsers/js/typescript-eslint-parser.js

```diff
@@ -0,0 +1,71 @@
+import React from 'react';
+import defaultParserInterface from './utils/defaultESTreeParserInterface';
+import pkg from 'typescript-eslint-parser/package.json';
+import SettingsRenderer from '../utils/SettingsRenderer';
+
+const ID = 'typescript-eslint-parser';
+
+const defaultOptions = {
+  range: true,
+  loc: false,
+  tokens: false,
+  comment: false,
+  tolerant: true,
+  useJSXTextNode: false,
+
+  ecmaFeatures: {
+    jsx: true,
+  },
+};
+
+const parserSettingsConfiguration = {
+  fields: [
+    'range',
+    'loc',
+    'tokens',
+    'comment',
+    'tolerant',
+    'useJSXTextNode',
+    {
+      key: 'ecmaFeatures',
+      title: 'ecmaFeatures',
+      fields: Object.keys(defaultOptions.ecmaFeatures),
```

Comment:
Sure, it looks like this:

![screen shot 2017-06-20 at 10 13 59 am](https://user-images.githubusercontent.com/6473925/27337751-59ca455c-55a1-11e7-837e-76f3d6caa6f2.png)